linera-execution/src/runtime.rs

```diff
@@ -33,12 +33,22 @@ use std::{
 /// Runtime data tracked during the execution of a transaction.
 #[derive(Debug, Clone)]
 pub(crate) struct ExecutionRuntime<'a, C, const WRITABLE: bool> {
+    /// The amount of fuel available for executing the application.
+    remaining_fuel: Arc<AtomicU64>,
+    /// the number of read
+    num_reads: Arc<AtomicU64>,
+    /// the total size being read
+    bytes_read: Arc<AtomicU64>,
+    /// the total size being write
```

Comment:
```suggestion
    /// The total size being written
```

(Or "that has been written so far"?)